ugvc/cnv/CNV_eval.py

```diff
@@ -0,0 +1,162 @@
+import sys
+import argparse
+import pandas as pd
+import os
+import warnings
+warnings.filterwarnings('ignore')
```

Comment:
Which warnings do you suppress? Pandas - could be nice to suppress in a more specific way